src/components/CorporateCampaign/CampaignLoanGridDisplay.vue

```diff
@@ -204,10 +207,12 @@ export default {
 				this.loans = data.lend?.loans?.values ?? [];
 				this.totalCount = data.lend?.loans?.totalCount ?? 0;
 
-				this.$emit('update-total-count', this.totalCount);
-				this.checkIfPageIsOutOfRange(this.loans.length, this.pageQuery.page);
+				if (this.isVisible) {
+					this.$emit('update-total-count', this.totalCount);
+					this.checkIfPageIsOutOfRange(this.loans.length, this.pageQuery.page);
```

Comment:
This notice was showing up while on the carousel interface if previous paging has been used in the grid interface.

I think we could eventually sync the paging across both views...